docs-new/package.json

```diff
@@ -15,8 +15,8 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.14",
-    "@docusaurus/preset-classic": "2.0.0-beta.14",
+    "@docusaurus/core": "^2.0.0-beta.15",
+    "@docusaurus/preset-classic": "^2.0.0-beta.15",
```

Comment:
Please avoid changes in `package.json` and `*-lock.lock` files.

beta-15 has minor breaking changes, will upgrade this in base PR.